webots_ros2_tests/test/test_system_driver.py

```diff
@@ -87,7 +87,7 @@ def testLidar(self):
         def on_message_received(message):
             # There should be some scans hitting the box
             for value in message.ranges:
-                if abs(value - 0.6749) < 0.01:
+                if abs(value - 0.76) < 0.01:
```

Comment:
Why this change?

Reply:
Because we fixed the measurements of the `RangeFinder` and `Lidar`, and this value correspond to the exact space between the box and the device.